### PR TITLE
whisky: new cask

### PR DESCRIPTION
**Pull request by IsaacMarovitz**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

_In the following questions `<cask>` is the token of the cask you're submitting._

After making any changes to a cask, existing or new, verify:

- [x] The submission is for [a stable version](https://docs.brew.sh/Acceptable-Casks#stable-versions) or [documented exception](https://docs.brew.sh/Acceptable-Casks#but-there-is-no-stable-version).
- [x] `brew audit --cask --online <cask>` is error-free.
- [x] `brew style --fix <cask>` reports no offenses.

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://docs.brew.sh/Cask-Cookbook#token-reference).
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://docs.brew.sh/Acceptable-Casks#finding-a-home-for-your-cask).
- [x] `brew audit --new-cask <cask>` worked successfully.
- [x] `brew install --cask <cask>` worked successfully.
- [x] `brew uninstall --cask <cask>` worked successfully.

### CODE REVIEW

**Comment by IsaacMarovitz** [comment]
- Only pre-release versions of Whisky are available to far
- Whisky is signed with a Developer ID, it is not, however, notarized

**Comment by razvanazamfirei** [comment]
Hi @IsaacMarovitz, thanks so much for submitting this. The app looks great! 

Is there a reason why the app is signed but not notarized?
In terms of pre-releases, I see that all versions so far are marked as pre-releases. This is just general advice and not a requirement _per se_, but I wonder whether tagging releases as pre-release achieves much. When everything as a pre-release, nothing is.

<details>
<summary><a href="https://www.reddit.com/r/github/comments/4mwwjp/comment/d3z06it/">Quoting</a> this reddit post:</summary>

> The tag is largely for when you're preparing a post-1.0 release, e.g., you have some betas of 2.0 but you want most people to keep using the last 1.x version unless they're willing to deal with bugs. Before 0.x it's generally assumed that there aren't strong stability guarantees either way.
> If you're targeting a particular set of features for the "1.0" label, but you have stable/unstable versions before that, it may make sense to mark only some revisions as prereleases. For instance, you might be basically done with a good chunk of the [app] at 0.5, but you're reworking [a feature] because you think you designed it all wrong. It might make sense to mark e.g. 0.6 as a pre-release while you finish and debug the new [feature] code, but mark 0.5 and 0.5.1 as stable because the old [feature] code is complete and working.

</details>

I would suggest marking releases as such, but let me know if that's not feasible for some reason.

**Comment by IsaacMarovitz** [comment]
Notarizing requires enabling the hardened runtime, and since Whisky relies on Wine under the hood, I ran into some permissions errors using it before. We did recently restructure where Wine is installed on the user's system, so it's possible those issues have been resolved now. As for the pre-release marker, I can disable it.

File: Casks/whisky.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  end
+
+  auto_updates true
+  depends_on macos: ">= :ventura"
+
+  app "Whisky.app"
+
+  zap trash: [
+    "~/Library/Application Support/Whisky",
+    "~/Library/Containers/com.isaacmarovitz.Whisky",
```
[the remark this answers is not in the document]
Comment (UltraHDR) [reply Casks/whisky.rb:23]: AppCleaner suggested that

[the remark this answers is not in the document]
Comment (razvanazamfirei) [reply Casks/whisky.rb:23]: Thanks, @UltraHDR!

**Comment by IsaacMarovitz** [comment]
Next release will be notarized

**Comment by razvanazamfirei** [comment]
Thanks, @IsaacMarovitz!

**Comment by IsaacMarovitz** [comment]
@razvanazamfirei New release is out :D

**Comment by miccal** [comment]
Thank you @IsaacMarovitz.

**Review by miccal — APPROVED** [review]

**Pull request merged**